flow/core/kernel/kernel.py

```diff
@@ -83,9 +90,9 @@ def update(self, reset):
             specifies whether the simulator was reset in the last simulation
             step
         """
-        self.scenario.update(reset)
+        # self.scenario.update(reset)
         self.simulation.update(reset)
-        self.vehicle.update(reset)
+        # self.vehicle.update(reset)
```

Comment:
why?

Reply:
just a temporary fix until we merge the next wave of PRs